.github/workflows/cd-publish.yml

```diff
@@ -3,38 +3,98 @@ name: CD - Publish for Blazor / FluentUI
 on:
   workflow_dispatch
 
+env:
+  OUTPUT_PATH: ${{ github.workspace }}/.nupkg
+  NUGET_SOURCE: "https://api.nuget.org/v3/index.json"
+  DOTNET_VERSION: "5.0.5"
+  GITHUB_SOURCE: "https://nuget.pkg.github.com/Azure/index.json"
+  CONFIGURATION: Release
+
 jobs:
-  publish:
+  build job:
     name: setup, build, validate, and publish
     runs-on: windows-latest
     if: contains('nicholasrice,awentzel,eisenbergeffect,Ibrahimmaga', github.actor)
```

Comment:
make all of these lower case

Reply:
it's been working. why change it?

Reply:
because I'm confident it won't break it and it's consistent.

Reply:
You should mark as "Resolved conversation" after the change to lowercase is reflected in this PR. I'm not seeing the change yet.

Reply:
it is changed